lib/pages/dashboard/blocs/agenda_bloc.dart

```diff
@@ -56,8 +65,8 @@ class AgendaBloc extends Bloc<AgendaEvent, AgendaState> {
 
   @override
   Stream<AgendaState> mapEventToState(AgendaEvent event) async* {
+    yield AgendaLoading(state.events);
```

Comment:
What is this parameter for, what does it contain? I can't remember...

Reply:
I think it was to implement in the future an infinite scroll agenda that not loads all the events in one go.

Reply:
I'm not sure this is useful anymore.